site/content/docs/howto/supervisor/configure-supervisor-federationdomain-idps.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Configure Identity Providers (IDPs) on a FederationDomain
+description: Learn how to use one or more identity providers, and identity transformations and policies, on a FederationDomain.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: IDPs on FederationDomains
+    weight: 20
+    parent: howto-configure-supervisor
+---
+
+This guide explains how to associate one or more external identity providers (IDPs) with a FederationDomain.
+It also details how to configure identity transformations and identity policies for those identity
+providers.
+
+## Prerequisites
+
+This how-to guide assumes that you have already [installed the Pinniped Supervisor]({{< ref "install-supervisor" >}})
+and have already read the guide about how to [configure the Supervisor as an OIDC issuer]({{< ref "configure-supervisor" >}}).
```

Comment:
I do find [configure the Supervisor as an OIDC issuer](https://deploy-preview-1660--pinniped.netlify.app/docs/howto/supervisor/configure-supervisor/) a little disorienting as it links to `Required Configuration` in the menu.  I think either we should align the text with some kind of symmetry or say:

```
 read the guide about how to configure the Supervisor as an OIDC issuer via the [Required Configuration document](#)
```

Reply:
Let's consider reorganizing this new page and the pre-existing "Required Configuration" page in a future PR. For this PR, I just want to get the information about how to use the new features onto the site.